tests/Doctrine/Tests/Models/OneToOneInverseSideLoad/InverseSide.php

```diff
@@ -13,7 +13,7 @@
 
 /**
  * @Entity()
- * @Table(name="one_to_one_inverse_side_load_inverse")
+ * @Table(name="o2o_side_inverse")
```

Comment:
[Because of the maximum identifier length](https://docs.oracle.com/en/database/oracle/oracle-database/21/odpnt/EFCoreIdentifier.html#GUID-FA43F1A1-EDA2-462F-8844-45D49EF67607) for OracleDB 11